resolvers/npm_installer/Cargo.toml

```diff
@@ -43,13 +42,16 @@ serde.workspace = true
 serde_json.workspace = true
 sys_traits = { workspace = true, features = ["real"] }
 thiserror.workspace = true
-tokio.workspace = true
-tokio-util.workspace = true
 twox-hash.workspace = true
 url.workspace = true
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+sys_traits = { workspace = true, features = ["real", "wasm"] }
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+tokio.workspace = true
```

Comment:
This actually doesn't compile without this yet. I'll fix this up in a follow-up PR.